pulp_service/pulp_service/app/viewsets.py

```diff
@@ -181,3 +184,28 @@ def create(self, request):
 
         task = dispatch(dispatch_task, shared_resources=shared_resources, kwargs=kwargs)
         return OperationPostponedResponse(task, request)
+
+
+class TaskIngestionDispatcherView(APIView):
+
+    authentication_classes = []
+    permission_classes = []
+
+
+    def get(self, request=None, path=None, task_num=1):
+        if not settings.TEST_TASK_INGESTION:
+            raise PermissionError("Access denied.")
+
+        task_count = 0
+        start_time = datetime.now()
+        timeout = timedelta(seconds=25)
```

Comment:
I believe 25 should come from the client. It's possible in different environments we'll set the gunicorn timeout differently. This should be a required paramenter passed to the viewset.